docs/man/rpm-man-template.scd

```diff
@@ -0,0 +1,138 @@
+RPMCOMMAND(1)
+
+# NAME
+*rpmcmd* - Example rpm man page
+
+# SYNOPSIS
+Use the generic form for one-purpose commands:
+
+*rpmcmd* [options] [_ARGUMENT_] ...
+
+Use one entry per operation for multi-operation (aka mode) commands.
+Use {} for the operation switches:
+
+*rpmcmd* {*-b*|*--bardize*} [options] [bardize-options] [_ARGUMENT_] ...
+
+*rpmcmd* {*-f*|*--frobnize*} [options] [_ARGUMENT_] ...
+
+# DESCRIPTION
+Describe what *rpmcmd* command does in plain, understandable
+English. Command-line switches belong to *OPTIONS* and *OPERATIONS*
+as appropriate.
+
+Typographic conventions:
+- Use hard wrapping at 80 columns for all text
+- Use bold for literal values such as command name and options:
+*cmd* does X, with option *--do-this* does Y instead
+- Use bold when referring to section names
+- Use bold when referring to other man page names, eg *rpm*(8)
+- Use underline uppercase to denote user arguments: _ARGUMENT_
+- Use underline for literal file names: _hello-1.0-1.noarch.rpm_
+- Use alphabetical order whenever multiple things are listed and
+  there's no contextual reason to do otherwise
+- Put the short variant first when listing option aliases
+- Put an empty line before a new section/option/etc
+- Put text immediately below a section/option/etc (ie without an empty line)
+- Put optional elements into [] brackets
+- Do not emphasize separators like ",", "=", "[]", "<>", "|" etc
+
+When in doubt, consult *man-pages*(7).
+
+# OPERATIONS
+For multi-operation commands, describe each operation invocation
+using same notation as for *OPTIONS*. Delete this section if the
+command only has a single main operation.
+
+*-f*,
+*--frobnize*,
+	Describe frobnization in detail.
+	Multiple lines are the expectation.
+
+*-b*,
+*--bardize*
+	Describe bardization in detail.
+	Multiple lines are the expectation.
+
+# ARGUMENTS
+Describe positional _ARGUMENT_ placeholders from *SYNOPSIS* here.
+
+_ARGUMENT_
+	File of type X.
+
+# OPTIONS
+Describe common options and in this section.
+
+Typographic conventions:
+- Put option aliases on separate lines
+- Put option arguments with pre-determined values into <> (see below)
+
+See *rpm.common*(8) for the options common to all *rpm* executables.
+
+*-d*,
+*--do-this*
+	Do Y instead of default action.
+
+*-o* _ARG_,
+*--output*=_ARG_
```

Comment:
@ferdnyc's comments in the other thread made me realize we should actually name the placeholders by what they represent, i.e. `_FILE_` in this case. It makes referring to the argument more natural when describing it, too. 

For example:

Current style:
```
*--output*=_ARG_
        Use file _ARG_ for output.
```

Proposed style:
```
*--output*=_FILE_
        Use _FILE_ for output.
```

Reply:
If we go with the proposed style, we should also mention this briefly in the `# AGUMENTS` section.

Reply:
But `# ARGUMENTS` is for positional arguments, not option arguments :sweat_smile:

Reply:
Oh and okay, *that* goes for both kinds of arguments so it deserves an explicit mention.

Reply:
Clarified this in both the typographic conventions + arguments sections.